pages/1.9/release-notes/1.9.8/index.md

```diff
@@ -0,0 +1,235 @@
+- StatsD endpoint in every container for forwarding metrics to the DC/OS metrics service. This service is what exposes the HTTP API.
+- Any metric sent to STATSD_UDP_HOST/PORT is available in the HTTP API’s `/container/<container_id>/app` endpoint.
+
+For more information, see the [documentation](/1.9/metrics/).
+
+### Tool for Troubleshooting Service Deployment Failures
+
+- The new service deployment troubleshooting tool allows you to find out why your applications aren’t starting from the GUI and CLI.
+
+  ![Service deploy GUI](/1.9/img/dcos-offers.png)
+
+### Improved GUI
+
+- New look and feel and improved navigation.
+
+  ![New GUI](/1.9/img/dcos-dash.png)
+
+- Usability improvements to the service create workflow.
+
+  ![Improved GUI](/1.9/img/dcos-create.png)
+
+## Networking Services
+
+- CNI support for 3rd party CNI plugins.
+- Performance improvements across all networking features.
+
+## Security and Governance
+
+- DC/OS Identity and Access Management (IAM) highlights:
+
+  - LDAP group import: support importing `posixGroup` objects according to [RFC2307](https://www.ietf.org/rfc/rfc2307) and [RFC2307bis](https://tools.ietf.org/html/draft-howard-rfc2307bis-02), and ensure compatibility with [FreeIPA](https://www.freeipa.org) and [OpenLDAP](https://www.openldap.org/).
+  - SAML 2.0: ensure that the authentication flow works against [Shibboleth](https://www.shibboleth.net/products/identity-provider/) and improve compatibility with a wide range of identity provider configurations.
+  - OpenID Connect: ensure that the authentication flow works against [dex](https://github.com/coreos/dex) as well as against [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/), and allow for customizing the identity provider certificate verification in back-channel communication. Enhance configuration validation for a better user experience.
+
+- DC/OS CLI highlights:
+
+  - Support single sign-on authentication via OpenID Connect and SAML 2.0 against the DC/OS IAM.
+  - Support authentication with service account credentials.
+
+- Introduce various secrets improvements (for more information, see the [secrets documentation](/1.9/security/ent/secrets/)).
+- Security hardening across the platform, including Mesos, Marathon, and Admin Router.
+
+## Developer Services
+
+- Jenkins
+
+    - The Jenkins DC/OS service will now work with DC/OS clusters in strict mode. <!-- (Enterprise Only) -->
+    - Marathon plugin now supports service accounts, allowing easy automated and secure deploys to DC/OS clusters. <!-- (Enterprise Only) -->
+
+## Other Improvements
+
+### DC/OS Internals
+
+- Update DC/OS internal JDK to 8u112 for security [fixes](http://www.oracle.com/technetwork/java/javase/2col/8u112-bugfixes-3124974.html).
+- Update DC/OS internal Python from 3.4 to 3.5.
+- The `dcos_generate_config.ee.sh --aws-cloudformation` command will now determine the region of the s3 bucket automatically, preventing region mistakes.
```

Comment:
Enterprise: add [enterprise type="inline" size="small" /]

Reply:
Added Enterprise tags.

Reply:
I think we should probably remove the `.ee` and then also remove the enterprise label.

I assume this statement holds true for both, DC/OS and DC/OS Enterprise.

Reply:
@anithamuthu79 looks like we forgot to remove the `.ee` here.